ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/productdetails/clientlibs/js/detailsofproduct.js

```diff
@@ -0,0 +1,72 @@
+$(document).ready(function () {
+	$('.quantity').val("1");
+	$('.product-details-cmp__prduct-price-sign').css("display", "none");
+	$('.product-details-cmp--no-product').css("display", "none");
+	$('.cmp-button').addClass("btn-product-card-disabled");
+	$(".cmp-button").attr('disabled', 'disabled');
+	const firstSizeSelected = $(".product-details-cmp__product-size--item:first-child").text();
+	$('.select-size').text(firstSizeSelected);
+	$(".product-details-cmp__product-size--item:first-child").addClass("selected-size");
+	const xhttp = new XMLHttpRequest();
+	xhttp.onreadystatechange = function () {
+		if (this.readyState == 4 && this.status == 200) {
```

Comment:
@Rajamouli-hcl  @AyushHCL  JS error still popping up on browser console.
![image](https://user-images.githubusercontent.com/63295621/82981143-dc094900-a008-11ea-80c9-9d2c2c0679dc.png)

Reply:
Hi Rakesh
As discussed this error message is from another component..not from PD page.

Reply:
@AyushHCL : can you pl check on the issue .. QA can raise a bug for that componet to fix

Reply:
okay Rishab..

Reply:
fixed js issue